python37:
  image: registry.gitlab.com/potato-oss/google-cloud/django-gcloud-connectors/python37:latest
  stage: test
  before_script:
    - mkdir .reports
  script: tox -e py37
  artifacts:
    reports:
      junit: .reports/django-tests.xml

python36:
  image: registry.gitlab.com/potato-oss/google-cloud/django-gcloud-connectors/python36:latest
  stage: test
  before_script:
    - mkdir .reports
  script: tox -e py36
  artifacts:
    reports:
      junit: .reports/django-tests.xml

flake8:
  # Currently, we have a few things failing, so allowing it to fail.
  # We should remove this once it's fixed
  allow_failure: true
  image: registry.gitlab.com/potato-oss/google-cloud/django-gcloud-connectors/python37:latest
  before_script:
    - mkdir .reports
  script: tox -e flake8 -- --format junit-xml --output-file=".reports/flake8.xml"
  artifacts:
    reports:
      junit: .reports/flake8.xml